.github/workflows/compatibility.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           - woocommerce: '4.8.0'
             wordpress:   '5.7'
             gutenberg:   '11.4.0' # The latest version supporting WP 5.6.
-            php:         '7.1' # TODO This should be 7.0. Pending to fix in issue #2970.
+            php:         '7.0' # Minimum supported PHP version
```

Comment:
Could you clean up the other line in this file too? 

https://github.com/Automattic/woocommerce-payments/blob/cf4f777e16c146c18bc5823e9dae76474c67fe60/.github/workflows/compatibility.yml#L43-L43

Reply:
Fixed.